### ISSUE TITLE
meetings-stage-view sample not working.

### ISSUE THREAD

**Issue by Nivedipa-MSFT** [OfficeDev/Microsoft-Teams-Samples#1367]
[meetings-stage-view](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/csharp) sample not working.

Getting below error:

![image](https://github.com/user-attachments/assets/339c24d9-0452-4df1-a4e1-baa7e40eb8fc)

**Comment by joylin2024** [comment]
Thank you for your request. I have reviewed the SharingProtocol.ScreenShare, and here is the relevant sample code/reference:

// Share the content in view-only screen sharing mode.
      const shareSpecificAppContentScreenShare = (partName) => {
          var appContentUrl = "";
          microsoftTeams.app.getContext().then((context) => {
              appContentUrl = partName == 'todo' ? `${window.location.origin}/todoView?meetingId=${context.meeting.id}` : partName == 'doing' ? `${window.location.origin}/doingView?meetingId=${context.meeting.id}` : `${window.location.origin}/doneView?meetingId=${context.meeting.id}`;
              microsoftTeams.meeting.shareAppContentToStage((err, result) => {
                  if (result) {
                      // handle success
                      console.log(result);
                  }

                  if (err) {
                      // handle error
                      alert(JSON.stringify(err))
                  }
              }, appContentUrl,
                  // Optional shareOptions with sharingProtocol set to ScreenShare
                  {
                      sharingProtocol: microsoftTeams.meeting.SharingProtocol.ScreenShare
                  });
          });
      };

 https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage?tabs=desktop%2Capp-content%2Cshare-app-content#screen-share-content-to-meetings

**Comment by Nivedipa-MSFT** [comment]
We are checking this with sample team.

**Comment by joylin2024** [comment]
Thanks for checking with the sample team. Looking forward to the updates.

**Comment by ChetanSharma-msft** [comment]
@Pawank-MSFT, @Harikrishnan-MSFT - Kindly help to check and fix this issue as soon as possible.

**Comment by Nivedipa-MSFT** [comment]
@joylin2024 - PR has raised and merged. PR link is [here](https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1375). Could you please test this sample?

**Comment by joylin2024** [comment]
Yes. I'll run the test and get back to you with the results soon.

**Comment by joylin2024** [comment]
I attempted to run the code, but I keep encountering errors.

**Comment by Nivedipa-MSFT** [comment]
@joylin2024 - Could you please share the error that you are getting?

**Comment by joylin2024** [comment]
Here is the error:
![Screenshot 2024-08-21 103443](https://github.com/user-attachments/assets/617c12bc-a705-4887-a93b-dc7b7be8f2c7)

**Comment by joylin2024** [comment]
I attempted to run the HelloWorld In-meeting app, but encountered the following error:

![Screenshot 2024-08-21 094737](https://github.com/user-attachments/assets/f2066e66-305b-4160-9f0b-12e39fde05e0)

**Comment by ChetanSharma-msft** [comment]
> I attempted to run the HelloWorld In-meeting app, but encountered the following error:
> 
> ![Screenshot 2024-08-21 094737](https://private-user-images.githubusercontent.com/164919859/359968455-f2066e66-305b-4160-9f0b-12e39fde05e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyNTI1NTcsIm5iZiI6MTcyNDI1MjI1NywicGF0aCI6Ii8xNjQ5MTk4NTkvMzU5OTY4NDU1LWYyMDY2ZTY2LTMwNWItNDE2MC05ZjBiLTEyZTM5ZmRlMDVlMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyMVQxNDU3MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNTRlMmI3YzdkYTE3ZTllMmMxYTAxYjk0YzQ3MzkyY2QwYjdkYjU2YjNmNjdmMzE4OTFiMTQwNjNmNDZhOTY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zcLLNW6icjojPIjiadoh2cFllFvMBmiMu8U0va9Ht1A)

@joylin2024 - Could you please share the steps followed?
We will try to run and let you know the updates.

**Comment by joylin2024** [comment]
I found this sample in Visual Studio Code.

1. I clicked the 'Create' button.
2. I ran the project.
3. A window briefly flashed on the screen and then disappeared.

**Comment by joylin2024** [comment]
I noticed that the "Dice Roller in meeting" sample isn't working. Could you please check it out as well?

**Comment by Nivedipa-MSFT** [comment]
> I attempted to run the HelloWorld In-meeting app, but encountered the following error:
> 
> ![Screenshot 2024-08-21 094737](https://private-user-images.githubusercontent.com/164919859/359968455-f2066e66-305b-4160-9f0b-12e39fde05e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyNjMyODAsIm5iZiI6MTcyNDI2Mjk4MCwicGF0aCI6Ii8xNjQ5MTk4NTkvMzU5OTY4NDU1LWYyMDY2ZTY2LTMwNWItNDE2MC05ZjBiLTEyZTM5ZmRlMDVlMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyMVQxNzU2MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZjYzNDhlNDYxZWI2YzM4NDg4ZTU4N2ZkYTcyZDE2ZTliNThhODFhY2Q4YmJiMTIzODUxMzU5Mjc0MjZkZmI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.DTVRD4W_0g1rdt2Rt6QjdIlOM2NTGAGEeNWZBABuKe0)

@joylin2024 - We tried to setup this application in VS Code however we could not find the application. Could you please let us know from which repo you got the sample reference?

**Comment by Nivedipa-MSFT** [comment]
@joylin2024 - Could you please share the details of your issue by sending an email to microsoftteamsdev@microsoft.com so that we can sync on a call?

**Comment by joylin2024** [comment]
OK. I will send an email to microsoftteamsdev@microsoft.com.

**Comment by Nivedipa-MSFT** [comment]
As discussed on the call, you need to explore the following two scenarios:
Share to stage with screen share.
Screen share with pop-out the share to stage view.
We have received input from the engineering team stating that these scenarios are not supported via the Teams JS library yet.

**Comment by joylin2024** [comment]
Ok.  Thanks!
